src/test/java/au/org/ala/names/search/BiocacheMatchTest.java

```diff
@@ -434,7 +434,7 @@ public void testSubSpeciesMarker7()  {
             String name = "Eucalyptus leucoxylon ssp.";
             cl.setScientificName(name);
             MetricsResultDTO metrics = searcher.searchForRecordMetrics(cl, true);
-            assertEquals("http://id.biodiversity.org.au/name/apni/237881", metrics.getResult().getLsid());
```

Comment:
This original line is what Travis is reporting as the actual result, and failing the test currently.